.pipelines/ci.yml

```diff
@@ -383,56 +383,50 @@ jobs:
     pool:
       vmImage: 'macOS-12'
     timeoutInMinutes: 120
-    variables:
-      OnnxruntimeDirectory: cmake/external/onnxruntime
     steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: "3.9"
+          addToPath: true
+          architecture: "x64"
+        displayName: "Use Python 3.9"
+
       - script: brew install coreutils ninja
         displayName: Install coreutils and ninja
 
-      - script: |
-          mkdir cmake/external
-          cd cmake/external
-          git clone https://github.com/microsoft/onnxruntime.git
-          cd onnxruntime
-          git checkout v1.12.1
-        displayName: Download Onnxruntime
+      - template: templates/with-running-android-emulator-steps.yml
```

Comment:
nit: 'with-running' in the name sounds like the emulator needs to be started previously. would 'run-with-android-emulator-steps' be slightly clearer that it will take care of emulator startup?

Reply:
yeah I think that's a better name